sway/input/seat.c

```diff
@@ -1315,11 +1317,6 @@ void seat_set_focus_layer(struct sway_seat *seat,
 		return;
 	}
 	assert(layer->surface->mapped);
-	if (layer->current.layer >= ZWLR_LAYER_SHELL_V1_LAYER_TOP &&
```

Comment:
Seems like we're loosing this condition here?

IOW, exclusive keyboard layers under the toplevels would never get focus before, but now they do.

Reply:
Why do we have this condition in the first place?

Reply:
That's a decision we made when implementing layer-shell support. The motivation is that if a particular layer has focus, a lower level layer cannot get focus. So if a toplevel has focus, background/bottom cannot get focus.